packages/manager/cypress/e2e/core/cloudpulse/edit-user-alert.spec.ts

```diff
@@ -0,0 +1,380 @@
+          .should('have.value', rule[key]);
+      });
+    });
+  };
+
+  it('should correctly display the details of the alert in the Edit Alert page', () => {
+    cy.visitWithLogin(`/monitor/alerts/definitions/edit/${service_type}/${id}`);
+    cy.wait('@getAlertDefinitions');
+
+    // Verify form fields
+    cy.findByLabelText('Name').should('have.value', label);
+    cy.findByLabelText('Description (optional)').should(
+      'have.value',
+      description
+    );
+    cy.findByLabelText('Service')
+      .should('be.disabled')
+      .should('have.value', 'Databases');
+    cy.findByLabelText('Severity').should('have.value', 'Severe');
+
+    // Verify alert resource selection
+    cy.get('[data-qa-alert-table="true"]')
+      .contains('[data-qa-alert-cell*="resource"]', 'database-3')
+      .parents('tr')
+      .find('[type="checkbox"]')
+      .should('be.checked');
+
+    // Verify alert resource selection count message
+    cy.get('[data-qa-notice="true"]')
+      .find('p')
+      .should('have.text', '1 of 5 resources are selected.');
+
+    // Assert rule values 1
+    assertRuleValues(0, {
+      aggregationType: 'Average',
+      dataField: 'CPU Utilization',
+      operator: '=',
+      threshold: '1000',
+    });
+
+    // Assert rule values 2
+    assertRuleValues(1, {
+      aggregationType: 'Average',
+      dataField: 'Memory Usage',
+      operator: '=',
+      threshold: '1000',
+    });
+
+    // Verify that tooltip messages are displayed correctly with accurate content.
+    ui.tooltip.findByText(METRIC_DESCRIPTION_DATA_FIELD).should('be.visible');
+    ui.tooltip.findByText(SEVERITY_LEVEL_DESCRIPTION).should('be.visible');
+    ui.tooltip.findByText(EVALUATION_PERIOD_DESCRIPTION).should('be.visible');
+    ui.tooltip.findByText(POLLING_INTERVAL_DESCRIPTION).should('be.visible');
+
+    // Assert dimension filters
+    const dimensionFilters = [
+      { field: 'State of CPU', operator: 'Equal', value: 'User' },
+    ];
+
+    dimensionFilters.forEach((filter, index) => {
+      cy.get(
+        `[data-qa-dimension-filter="rule_criteria.rules.0.dimension_filters.${index}-data-field"]`
+      )
+        .should('be.visible')
+        .find('input')
+        .should('have.value', filter.field);
+
+      cy.get(
+        `[data-qa-dimension-filter="rule_criteria.rules.0.dimension_filters.${index}-operator"]`
+      )
+        .should('be.visible')
+        .find('input')
+        .should('have.value', filter.operator);
+
+      cy.get(
+        `[data-qa-dimension-filter="rule_criteria.rules.0.dimension_filters.${index}-value"]`
+      )
+        .should('be.visible')
+        .find('input')
+        .should('have.value', filter.value);
+    });
+
+    // Verify notification details
+    cy.get('[data-qa-notification="notification-channel-0"]').within(() => {
+      cy.get('[data-qa-channel="true"]').should('have.text', 'Channel-1');
+      cy.get('[data-qa-type="true"]').next().should('have.text', 'Email');
```

Comment:
```suggestion
      cy.get('[data-qa-type]').next().should('have.text', 'Email');
```

same here, can we try and see, this looks good more with me